tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml

```
@@ -58,8 +58,8 @@ content_types:
         help_text: Use the publication page for books, journal articles, or other citable work.
         category: General
         id: pub
-        title_format: "[publication_title]"
-        url_format: "pub/[TripalEntity__entity_id]"
+        title_format: "[pub_title]"
+        url_format: "publication/[TripalEntity__entity_id]"
```

Reverts this change back to `publication` instead of `pub`.  We're not sure how this keeps popping up in our PRs, but this matches the fix @laceysanderson made earlier.

```suggestion
        title_format: "[publication_title]"
        url_format: "pub/[TripalEntity__entity_id]"
```

---

@spficklin why did you suggest this? This has now broken the title for publications.
Under Available tokens it says `[pub_title]` instead of `[publication_title]`

---

@laceysanderson would like to fix this after this gets merged so she can marke sure this doesn't keep happening.  So let's restore it back the way you had it.